x-pack/docs/en/security/backup-security-configuration.asciidoc

```diff
@@ -0,0 +1,263 @@
+--------------------------------------------------
+curl -X GET -u elastic "localhost:9200/_cluster/settings?pretty&flat_settings" | jq '.persistent | with_entries(select(.key|startswith("xpack.security")))'
+--------------------------------------------------
+
+You can store the output of this together with the other configuration files.
+
+NOTE: Transient settings are not considered for backup.
+
+--
+
+[float]
+[[backup-security-repos]]
+===== Controlling access to the backup repository
+
+If a third party reads the snapshot and exposes the contents of the `.security`
+index, it does not mean you have compromised cluster security. An attacker
+would still need to brute force cryptographic hashes before they can
+impersonate users or use API keys. However, every document in the security
+index is sensitive to changes. Modifications to that index have the potential
+to compromise the integrity of all the data in the cluster. To prevent this
+situation:
+
+* Snapshot the `.security` index in a dedicated repository, where read and write
+access is strictly restricted and audited.
+* If there are indications that the snapshot has been read, change the passwords
+of the users in the native realm and revoke API keys.
+* If there are indications that the snapshot has been tampered with, do not
+restore it. There is currently no option for the restore process to detect
+malicious tampering.
+
+The same recommendations apply to storing backups for configuration files.
+However, use special caution when you store the `elasticsearch.keystore` and
+private key file backups. Currently, the secure store is *not* password
+protected and any read access can compromise the passwords of external systems
+and private keys.
+
+[float]
+[[restore-security-configuration]]
+==== Restoring security configuration
+
+NOTE: You can restore a snapshot of the `.security` index only if it was created
+in the same major release version. The last minor version of every major release
+can convert and read both versions of the index.
+
+To restore your security configuration from a backup, log in to one of the node
+hosts, navigate to {es} installation directory, and follow these steps:
+
+. Make sure the repository holding the `.security` snapshot is installed.
++
+--
+[source,js]
+--------------------------------------------------
+GET /_snapshot/my_backup
+--------------------------------------------------
+// CONSOLE
+--
+
+. Add a new user with the `superuser` built-in role to the file based realm.
++
+--
+For example, create a user named `jacknich`:
+[source,shell]
+--------------------------------------------------
+bin/elasticsearch-users useradd jacknich -p password -r superuser
+--------------------------------------------------
+--
+
+. Using the previously created user, delete the existing `.security-6` or
+`.security-7` index.
++
+--
+[source,shell]
+--------------------------------------------------
+curl -u jacknich -X DELETE "localhost:9200/.security-7"
```

Comment:
Why are you electing to put this in `shell` rather than `console`?

Reply:
This doesn't look like a Kibana/GUI type of operation to me. And the user already has a `shell` on the local node, so this might come easier to him. But it's prob my brain that associates hackish operations like these with cmd lines.